frontend/src/components/tournament/CreatePools.js

```diff
@@ -0,0 +1,225 @@
+
+const allTeamsName = "All";
+
+/**
+ * @param {object} props
+ * @param {object} props.team
+ * @param {int} props.team.seed
+ * @param {str} props.team.name
+ * @returns
+ */
+const TeamInfo = props => (
+  <div class="p-2 text-start flex gap-4 cursor-grab text-sm">
+    <div class="basis-1/6 text-center text-white dark:text-white-400 rounded-xl border-b bg-green-400 border-green-200 dark:bg-green-800 dark:border-green-700">
+      {props.team.seed}
+    </div>
+    <div class="basis-5/6">{props.team.name}</div>
+  </div>
+);
+
+/**
+ * @param {object} props
+ * @param {{seed: int, name: str}} props.team
+ */
+const Draggable = props => {
+  const draggable = createDraggable(props.team.seed);
+  return (
+    <div
+      use:draggable
+      class="text-gray-500 dark:text-gray-400 my-2 rounded-lg bg-gray-200 border-gray-300 border-b dark:bg-gray-800 dark:border-gray-700"
+    >
+      <TeamInfo team={props.team} />
+    </div>
+  );
+};
+
+/**
+ *
+ * @param {object} props
+ * @param {str} props.id
+ * @param {str} props.name
+ * @param {{seed: int, name: str}[]} props.teams
+ */
+const Droppable = props => {
+  const droppable = createDroppable(props.id);
+  return (
+    <>
+      <div
+        use:droppable
+        class="flex flex-col justify-items-center justify-start gap-y-2 content-center h-full p-4 rounded-xl dark:text-gray-400 bg-white text-center select-none border dark:border-gray-700"
+        classList={{
+          "ring-4 ring-blue-500": droppable.isActiveDroppable,
+          "justify-between dark:bg-gray-700": props.id !== allTeamsName,
+          "dark:bg-gray-900": props.id === allTeamsName
+        }}
+      >
+        {/* bg-gray-200 dark:bg-gray-700 border-b dark:border-gray-600 rounded-b-xl */}
+        <h4 class="text-center text-white p-2 justify-self-start">
+          <Show when={props.id !== allTeamsName} fallback={"All teams"}>
+            Pool - {props.name}
+          </Show>
+        </h4>
+        <div class="">
+          <For each={props.teams}>{team => <Draggable team={team} />}</For>
+        </div>
+        <Show when={props.id !== allTeamsName}>
+          <button
+            class="justify-self-end p-2 text-sm font-normal text-white rounded-lg bg-red-700 hover:bg-red-800 dark:bg-red-500/75 dark:hover:bg-red-700/75"
+            onClick={() => props.removePool(props.id)}
+          >
+            Remove pool
+          </button>
+        </Show>
+      </div>
+    </>
+  );
+};
+
+const CreatePools = props => {
+  const [enteredPoolName, setEnteredPoolName] = createSignal("");
+  const [draggedTeam, setDraggedTeam] = createSignal({});
+  const [draggableStartPool, setDraggableStartPool] = createSignal("");
+
+  const [pools, setPools] = createStore({});
+
+  createEffect(() => {
+    let allTeams = [];
+    props.allTeams.forEach((teamId, idx) =>
+      allTeams.push({
+        seed: idx + 1,
+        name: props.teamsMap[teamId]
+      })
+    );
+
+    setPools(allTeamsName, allTeams);
```

Comment:
Yo this will give wrong result when we have already created a pool, submitted and we refresh the page.

We will need to remove the teams which are already in the submitted pools.

Also well need to create the pools that were submitted before.. currently they will not show at all if we refresh after submitting

Reply:
Yeah i've fixed this now. 
@Joe2k: if tournament is still in Draft, we should be able to edit the pools even after submitting right ?

Reply:
yep so this is what we were talking in #180 

So can you pick those changes also in this PR?